x-pack/plugins/security_solution/cypress/screens/common/page.ts

```diff
@@ -8,3 +8,5 @@
 export const PAGE_TITLE = '[data-test-subj="header-page-title"]';
 
 export const NOT_FOUND = '[data-test-subj="notFoundPage"]';
+
+export const LOADING_SPINNER = '.euiLoadingSpinner';
```

Comment:
@dhurley14 , Actually I see that `LOADING_SPINNER` exists here as well. Do you think we can reuse existing one ( but I agree that this location is much better for loading spinner)

https://github.com/elastic/kibana/blob/95f7064861c52d5600c6bd9df47fdba6922814e7/x-pack/plugins/security_solution/cypress/screens/hosts/host_risk.ts#L10

Reply:
@MadameSheema added this in her commit here: 173b494805711b4e59744d998893488000653f82

I'll leave that up to her!